docs/lang/articles/basic/field.md

```diff
@@ -3,100 +3,137 @@ sidebar_position: 3
 ---
 
 # Fields
+Taichi fields are used to store data.
+In general, fields are used as global data containers that can be read and written from both the Python scope and the Taichi scope.
```

Comment:
Maybe we can remove `used as` here? i.e., `fields are global data containers...`